src/helper/serve.utils.ts

```diff
@@ -119,6 +119,8 @@ export async function startServer({ domain, host, isSSR, port }) {
 		return res.end()
 	})
 	app.get('/*', async (req, res) => {
+		const BUNDLE_PATH = path.join(process.cwd(), path.join('.fdk', 'dist', 'themeBundle.common.js'));
```

Comment:
Will it require a manual reload once the build is completed? @Shivraj0

Reply:
It will auto reload once build is ready with the existing flow, no need to trigger reload here.